js/index.js

```diff
@@ -14,9 +14,64 @@ function bindToForm(form) {
 	update()
 
 	form.addEventListener('change', update)
-	form.addEventListener('keyup', update)
+	form.addEventListener('keyup', function(e) {
+		handleDynamicForms(e, update)
+	})
 }
 
+function handleDynamicForms(e, next) {
+
+	function setCommuneFetchStatus(text) {
```

Comment:
Why declare this function inside the other one? Are you capturing any of the inner variables? If not, avoid creating a new `Function` object on each run of `handleDynamic Forms`, especially considering it is a user input event handler.